frontend/src/components/SignIn.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import axios from 'axios';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import FormControl from '@material-ui/core/FormControl';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import Checkbox from '@material-ui/core/Checkbox';
+import Input from '@material-ui/core/Input';
+import InputLabel from '@material-ui/core/InputLabel';
+import LockOutlinedIcon from '@material-ui/icons/LockOutlined';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import withStyles from '@material-ui/core/styles/withStyles';
+
+const styles = theme => ({
+  main: {
+    width: 'auto',
+    display: 'block', // Fix IE 11 issue.
+    marginLeft: theme.spacing.unit * 3,
+    marginRight: theme.spacing.unit * 3,
+    [theme.breakpoints.up(400 + theme.spacing.unit * 3 * 2)]: {
+      width: 400,
+      marginLeft: 'auto',
+      marginRight: 'auto'
+    }
+  },
+  paper: {
+    marginTop: theme.spacing.unit * 8,
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    padding: `${theme.spacing.unit * 2}px ${theme.spacing.unit * 3}px ${theme
+      .spacing.unit * 3}px`
+  },
+  avatar: {
+    margin: theme.spacing.unit,
+    backgroundColor: theme.palette.secondary.main
+  },
+  form: {
+    width: '100%', // Fix IE 11 issue.
+    marginTop: theme.spacing.unit
+  },
+  submit: {
+    marginTop: theme.spacing.unit * 3
+  }
+});
+
+class SignIn extends React.Component {
+  state = {
+    username: '',
+    password: ''
+}
+
+  InputHandler = event => {
+    event.preventDefault();
+    const target = event.target;
+    this.setState({ [target.name]: target.value });
+  };
+
+  SubmitHandler = event => {
+    event.preventDefault();
+    console.log(this.state);
+    const credentials = this.state;
+    const endpoint = 'https://league-management.herokuapp.com/auth/login';
+    axios
+      .post(endpoint, credentials)
+      .then(res => {
+        localStorage.setItem('jwt', res.data.token);
+        window.location.href = 'https://leaguemanagement.netlify.com/dashboard';
```

Comment:
I haven't used window.location.href before, but I see what it is doing. We could also handle this via react router instead with a Redirect to keep everything working with react router. If we decide to go down that route, we can handle it when we refactor later on.